presto-hive/src/main/java/io/prestosql/plugin/hive/InternalHiveConnectorFactory.java

```diff
@@ -152,4 +162,12 @@ public static Connector createConnector(String catalogName, Map<String, String>
                     classLoader);
         }
     }
+
+    private static Module defaultEventListenerModule()
+    {
+        return binder -> {
+            newSetBinder(binder, EventListener.class)
+                    .addBinding().toInstance(new EventListener() {});
```

Comment:
There's no need to create a no-op listener. The default implementation of `Connector.getEventListeners()` returns an empty set, so empty iterable is legal. We just need to create the set binder.